designs/virtualization/4.5/Scheduling/Firmware-config/Firmware-config.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: virtualization
+version: 4.5
+---
+
+# Firmware Config
+
+Exposing Firmware Config for VM.
+Firmware is placed within the scheduling section. It is related to CPU, and shoudn't be seperated from the rest of the CPU features. Because of that, it should be located next to other CPU features. Also, it affects performance, so it is related to "Resource Requirements".
+
+![VM page](img/Firmware-0-0.jpg)
+Firmware Config is placed in the Scheduling sction of the VM details tab.
+The default value of this field is "Default"
+
+![Config modal](img/Firmware-1-0.jpg)
+This modal is composed of three fields:
+- Bootload (Selection): BIOS (Default) and EFI
+- Serial (string): [The system-serial-number in SMBIOS]
+- UUID ( string): [UUID reported by the vmi bios. Defaults to a random generated UID]
+
+![Config modal](img/Firmware-1-1.jpg)
```

Comment:
Are these the correct field level help icons? I think we use something different elsewhere but honestly not sure if we have the right one in the console today. We are inconsistent. @mceledonia mind helping us with this one? fyi @rachael-phillips I know you recently used field level help icons in your designs as well. We also use them today in the console for fields on the environment tab of deployments and other resources.

Reply:
I didn't find an example for help icon in a modal, but I did find this:
<img width="476" alt="Screen Shot 2020-05-13 at 15 05 51" src="https://user-images.githubusercontent.com/48942365/81810400-8d997c00-952b-11ea-945d-1562d8cb4750.png">

Is this the latest icon? (I think we use help icon for models and underline for in-page)

Reply:
The underline I believe is reserved for the details page where we'd have a whole bunch adding clutter. We could argue here that an icon is better b/c there won't be anything with it (pencil icon) and there won't be a bunch on this page. 
The correct icon should be the (?) not the filled one you have here now.
https://fontawesome.com/icons/question-circle?style=regular